docs/configuration.md

```diff
@@ -9,6 +9,35 @@ menu:
     weight: 40
 ---
 
+# Default Directories
+
+Poetry uses these following default directories
+
+## Config Directory
+
+- Linux: `$XDG_CONFIG_HOME/$appname` or `~/.config/$appname`
+- Windows:`%USERPROFILE%\AppData\Local\$appauthor\$appname` (not roaming) or`%USERPROFILE%\AppData\Roaming\$appauthor\$appname` (roaming)
```

Comment:
```suggestion
- Windows: `%USERPROFILE%\AppData\Local\$appauthor\$appname` (not roaming) or `%USERPROFILE%\AppData\Roaming\$appauthor\$appname` (roaming)
```
What are `roaming` and `not roaming`? and `$appauthor` and `$appname`? Also, you forgot some spaces.

Reply:
I would actually rewrite these at `%APPDATA%` and `%LOCALAPPDATA%`.